reccmp/isledecomp/compare/core.py

```diff
@@ -260,22 +275,16 @@ def _match_array_elements(self):
         This step is necessary e.g. for `0x100f0a20` (LegoRacers.cpp).
         """
 
+        dataset = {}
+
         # Helper function
         def _add_match_in_array(
             name: str, type_id: str, orig_addr: int, recomp_addr: int
         ):
             # pylint: disable=unused-argument
             # TODO: Previously used scalar_type_pointer(type_id) to set whether this is a pointer
```

Comment:
I think this TODO can be removed

Reply:
I think I'll keep it until the next round. We may want to add type info to the database for these entries